cekit/template_helper.py

```diff
@@ -50,3 +50,24 @@ def ports(self, available_ports):
                 port_list.append(p.get('value'))
 
         return port_list
+
+    def artifacts(self, inarts):
```

Comment:
this is only necessary because the "docgen" action is being handled before the module initialisation (in fact, the module initialisation never happens). If the module init had happened, the in-memory modules would have missing name fields calculated.